rust/cbindgen.toml

```diff
@@ -86,8 +86,6 @@ include = [
     "FtpRequestCommand",
     "FtpStateValues",
     "FtpDataStateValues",
-    "SCSigTableAppLiteElmt",
```

Comment:
TODO afterwards remove `SCSigTableAppLiteElmt ` definition from rust and use the suricata_sys one